packages/control/lib/metrics.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const RuntimeApiClient = require('./runtime-api-client')
+
+async function runtimeMetricsCommand () {
+  const client = new RuntimeApiClient()
+  const runtime = await client.getMatchingRuntime([])
+  const metrics = await client.getRuntimeMetrics(runtime.pid, { format: 'json' })
```

Comment:
Can you add an option to select the format?

Reply:
Done ✅